website/blog/2023-02-28-ferretdb-release-0-9-2.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "FerretDB v0.9.2 - Tigris Authentication support"
+slug: ferretdb-v-0-9-2
+authors: [alex]
+description: "We are delighted to announce the release of FerretDB version 0.9.2, which includes Tigris authentation, support for `$listIndexes`, `$addtoSet`, and more."
+image: /img/blog/ferretdb_v0.9.2.jpg
+tags: [release]
+date: 2023-02-28
+---
+
+We are delighted to announce the release of FerretDB version 0.9.2, which includes Tigris authentation, support for `$listIndexes`, `$addtoSet`, and more.
+
+![FerretDB v0.9.1 - Minor Release](/img/blog/ferretdb_v0.9.2.jpg)
+
+<!--truncate-->
+
+The past few weeks were exciting: our cooperation with the open source community is stronger than ever.
+The feedback of developers and service providers makes us confident that we are on the right track with creating a truly open source alternative to MongoDB.
+
+Before we go into all the exciting new features in this release, let’s recap FerretDB and what we’re striving to achieve.
+[FerretDB](https://www.ferretdb.io/) is an [open-source alternative to MongoDB](https://blog.ferretdb.io/5-database-alternatives-mongodb-2023/) that converts MongoDB drivers and protocols into SQL with a backend on PostgreSQL and Tigris.
+
+We are gradually increasing compatibility with MongoDB – including tools, drivers, and the entire ecosystem, so FerretDB can act as a fully open source drop-in replacement for it.
+Be sure to try it out!
+
+Here are some of the notable changes in our previous releases:
+
+* Basic support for aggregation pipelines
+* More filter queries pushed down to the backend for Tigris and PostgreSQL
+* Client TLS certificate validation
+* Support for raspberry pi
+* Basic telemetry service
+
+This above list is by no means exhaustive; it’s only a snippet of some of the recent additions.
+If you missed our previous releases, see the [complete list of all the updates](https://github.com/FerretDB/FerretDB/releases/) to FerretDB.
+We’ll be adding lots of new features as we approach the FerretDB GA.
+If you have a feature request, feedback, or discovered a bug, please don’t hesitate to [contact us](https://docs.ferretdb.io/#community).
+We’d love to hear from you!
+
+Here are the latest updates on this new version of FerretDB:
+
+## New Features
+
+Coming into this release is support for the `listIndexes` command.
+With this command, you can now access a collection’s indexes.
+Besides `listindexes` command, we’ve also added support for `$addToSet` update operator, meaning you can easily add elements to an array as long as the element does not previously exist.
+Let’s take a look at an example:
+
+```js
+db.states.insertOne({
+  _id: 1,
+  name: "California",
+  colors: ["blue", "gold", "red"]
+})
+```
+
+Use the `$addtoSet` operator to update the array with more elements.
```

Comment:
That example does not show what is the difference from `$push`:
* https://www.mongodb.com/docs/manual/reference/operator/update/push/
* https://www.mongodb.com/docs/manual/reference/operator/update/addToSet/